src/ExternalSearch.Providers.Bregg/BrregExternalSearchProvider.cs

```diff
@@ -240,20 +240,22 @@ public IEnumerable<Clue> BuildClues(ExecutionContext context, IExternalSearchQue
             if (resultItem.Data.BrregNumber == 0)
                 return null;
 
-            var code = GetOriginEntityCode(resultItem, request);
+            var clue = new Clue(request.EntityMetaData.OriginEntityCode, context.Organization);
+            PopulateMetadata(clue.Data.EntityData, resultItem, request, config);
 
-            var clue = new Clue(code, context.Organization);
-
-            PopulateMetadata(clue.Data.EntityData, resultItem, request);
-            clue.Data.EntityData.Codes.Add(request.EntityMetaData.OriginEntityCode);
+            var jobData = new BrregExternalSearchJobData(config);
+            if (!jobData.SkipEntityCodeCreation)
```

Comment:
I don't think this needs to be here as the re is similar code in PopulateMetadata

Reply:
done